.circleci/config.yml

```diff
@@ -34,7 +37,11 @@ commands:
             if [[ $CIRCLE_BRANCH == "master" ]] ; then
               echo "export TINYBIRD_PYTEST_ARGS='--report-to-tinybird '" >> $BASH_ENV
             fi
-            echo "export TINYBIRD_DATASOURCE=community_tests_circleci" >> $BASH_ENV
+            if [ -f aws_credentials.env ]; then
+              echo "export TINYBIRD_DATASOURCE=scheduled_tests_circleci" >> $BASH_ENV
```

Comment:
The name here should imply that it's not just a scheduled run, but a multiaccounts / multiregion test run.
Maybe something like `community_tests_circleci_ma_mr`?

Reply:
If you migrate this workflow to be parameterized instead of duplicated you would also be able to use the parameter here.
This would have the benefit that it does not rely on the order of execution. This code here currently only works if `prepare-pytest-tinybird` is executed _after_ `generate-random-aws-credentials`.